posthog/models/organization.py

```diff
@@ -43,6 +43,7 @@ class OrganizationUsageInfo(TypedDict):
     surveys: Optional[OrganizationUsageResource]
     rows_synced: Optional[OrganizationUsageResource]
     cdp_invocations: Optional[OrganizationUsageResource]
+    rows_exported: Optional[OrganizationUsageResource]
```

Comment:
wdyt about calling it `batch_rows_exported` so that it's more distinct from `rows_synced` and has a clearer connection to batch exports? or do you think it could be too limiting considering how it can evolve? feel free to decide.